BonusCalcListener.Tests/BonusCalcListener.Tests.csproj

```diff
@@ -5,8 +5,9 @@
     <IsTestProject>true</IsTestProject>
     <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
+    <RunAnalyzersDuringBuild>False</RunAnalyzersDuringBuild>

```

Comment:
I looked at this https://learn.microsoft.com/en-us/visualstudio/code-quality/disable-code-analysis?view=vs-2022 -Why has this been turned off?

Reply:
It's been turned off because we use SonarCloud for code quality scanning. The internal Microsoft Codeanalysis scanner and SonarCloud have different standards and expectations, which don't always align. SC is the accepted Hackney standard, so we'll use that